File: src/main/resources/static/js/service/ddinfoService.js
//服务层
app.service('ddinfoService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../ddinfo/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../ddinfo/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../ddinfo/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../ddinfo/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../ddinfo/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../ddinfo/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../ddinfo/search.do?page='+page+"&rows="+rows, searchEntity);
	}
    //搜索物流信息
    this.searchwl=function(wltype,wlinfo){
        return $http.post('../ddinfo/searchwl.do?wltype='+wltype+"&wlinfo="+wlinfo);
    }
    //更新订单状态
    this.updateState=function(entity){
        return $http.post('../ddinfo/updateState.do',entity);
    }
});

File: src/main/java/com/zzti/fruits/service/DdinfoService.java
package com.zzti.fruits.service;
import java.util.List;

import com.zzti.fruits.entity.PageResult;
import com.zzti.fruits.pojo.Ddinfo;
import com.zzti.fruits.pojogroup.OrderPoiParam;

import javax.servlet.http.HttpServletResponse;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface DdinfoService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Ddinfo> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Ddinfo ddinfo);
	
	
	/**
	 * 修改
	 */
	public void update(Ddinfo ddinfo);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Ddinfo findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Ddinfo ddinfo, int pageNum, int pageSize);

	/**
	 * 更新订单状态
	 * @param ddinfo
	 */
    void updateState(Ddinfo ddinfo);

    PageResult findPoiPage(OrderPoiParam orderPoiParam, int page, int rows);

    void excelPoiSearch(OrderPoiParam orderPoiParam, HttpServletResponse response);

    void shortageoOrderReport(HttpServletResponse response);
}
